### PR TITLE
render video

### PR DESCRIPTION
**Pull request by sorja**
part of #1015

### CODE REVIEW

File: client/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import { Global } from '@core/country'
+import { FRA } from '@core/assessment'
+import * as BasePaths from '@webapp/main/basePaths'
+import { Breakpoints } from '@webapp/utils/breakpoints'
+
+import { useCountries } from '@webapp/store/app'
+import { useGroupedRegions } from '@webapp/store/app/hooks'
+
+import { useI18n } from '@webapp/hooks'
```
Comment (sorja) [inline client/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx:14]: This file will be handled in next pr

File: client/pages/Landing/Introduction/AreaSelector/DropdownAreas/DropdownAreas.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import React, { useLayoutEffect, useRef } from 'react'
+import { Link } from 'react-router-dom'
+import { FRA } from '@core/assessment'
+// import PanEuropean from '@common/assessment/panEuropean'
+import { RegionCode } from '@core/country'
+import * as BasePaths from '@webapp/main/basePaths'
+import { useI18n } from '@webapp/hooks'
+import Icon from '@webapp/components/icon'
```
Comment (sorja) [inline client/pages/Landing/Introduction/AreaSelector/DropdownAreas/DropdownAreas.tsx:8]: This file will be handled in next pr

File: client/pages/Landing/Introduction/AreaSelector/SelectMobile/SelectMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import { useI18n } from '@webapp/hooks'
```
Comment (sorja) [inline client/pages/Landing/Introduction/AreaSelector/SelectMobile/SelectMobile.tsx:4]: This file will be handled in next pr

File: client/pages/Landing/Introduction/AreaSelector/areaSelector.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
+@import 'webapp/style/partials';
```
Comment (sorja) [inline client/pages/Landing/Introduction/AreaSelector/areaSelector.scss:1]: This file will be handled in next pr

File: client/pages/Landing/Introduction/Introduction.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import './introduction.scss'
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+// import AreaSelector from './AreaSelector'
```
Comment (sorja) [inline client/pages/Landing/Introduction/Introduction.tsx:5]: This file will be handled in next pr

File: client/styles/buttons.less [hunk 6]
```diff
@@ -0,0 +1,129 @@
+@import (reference) 'webapp/main/styles/config';
```
Comment (sorja) [inline client/styles/buttons.less:1]: This file will be handled in next pr

**Review by minotogna — APPROVED** [review]

**Pull request merged**